app/screens/snack_bar/index.tsx

```diff
@@ -28,6 +28,8 @@ import {makeStyleSheetFromTheme} from '@utils/theme';
 import {typography} from '@utils/typography';
 
 const SNACK_BAR_WIDTH = 96;
+const SNACK_BAR_HEIGHT = 56;
+const SNACK_BAR_BOTTOM_RATIO = 0.13;
```

Comment:
@avinashlng1080 care to explain this value?

Reply:
It's the ratio of the snackbar's bottom to the height of the device from the figma designs and it turns out to be 13%.